### PR TITLE
Editorial improvements

### PR DESCRIPTION
**Pull request by jtroynousky**
This PR adds three small features:

1. The ability to turn off all dates in a given liveblog -- checkbox located in the Liveblog post meta box
2. The ability to customize the default "Live Feed" feed title -- also in the Liveblog post meta box
3. The ability to add an arbitrary URL when a new entry to selected as a "key event". When the key event is clicked on from the Key Events list, a new window will open with the selected URL instead of the default behavior of jumping to the full entry.

### CODE REVIEW

File: classes/class-wpcom-liveblog-entry-key-events.php [hunk 1]
```diff
@@ -114,6 +115,14 @@ public static function add_key_event_json( $json, $entry ) {
 
 		$json['key_event'] = self::is_key_event($json['id']);
 
+		// Add key event URL if it exists
+		if ( $json['key_event'] ) {
+			$key_event_url = get_comment_meta( $json['id'], self::META_KEY_URL, true );
+			if ( ! empty( $key_event_url ) ) {
+				$json['key_event_url'] = $key_event_url;
```
Comment (markparolisi) [inline classes/class-wpcom-liveblog-entry-key-events.php]: should this be sanitized here, or does the escaping happen elsewhere?

**Review by hrkhal — CHANGES_REQUESTED** [review]
Please validate the URL in the PHP before saving to prevent,  otherwise all is good.

Heads up that we use custom templates for the AMP liveblogs, so you'll need to make changes in those to surface the feed title, hide dates and to display key event urls instead.

File: classes/class-wpcom-liveblog-entry-key-events.php (hunk 1 above)
Comment (jtroynousky) [reply classes/class-wpcom-liveblog-entry-key-events.php]: I do `esc_url` prior to saving: https://github.com/hrkhal/liveblog/pull/18/files#diff-d261b87cfa9200cd48f1e766e6497379R225

But there should really be escaping here, as well. I'll add it.

**Review by jtroynousky — COMMENTED** [review]
Thanks @hrkhal -- yup, I can make those changes on the feed side. I've also added the URL escaping to the JSON output (the sanitization takes place here: https://github.com/hrkhal/liveblog/pull/18/files#diff-d261b87cfa9200cd48f1e766e6497379R225)

**Pull request merged**